email/te-feedback-email.html

```diff
@@ -0,0 +1,66 @@
+<body>
+  <div style="color:red; font-weight:bold;">
+    <p>{{if(1.changed != 1.completed; "This is an updated submission.")}}</p>
+    <p>{{if(23.rowId; ; "Task “" + 1.task_id + "” is not allotted.")}}</p>
+    <p>
+      {{if(23.columnfcname != 1.name; " File is allotted to another FC - " + 23.columnfcname + ".")}}
+    </p>
+    <p>
+      {{if(23.columnstatus = "Done"; " Status of the task is Done. Submitted data is not saved into Task!")}}
+    </p>
+  </div>
+
+  <p>
+    <a
+      href="http://purebhakti.info/audioseva/form/track-editing-fidelity-check?token={{encodeURL(1.token)}}"
+      >Update TFC Submission</a
+    >
+  </p>
+
+  <p>Hare Krsna! Dandavat Pranams.</p>
+  <p>
+    Below are the details of the
+    <b>Track Editing Fidelity Check (TFC)</b> feedback.
+  </p>
+
+  <p><b>Please re-edit the file and upload it again.</b></p>
+
+  <p style="color: red;font-weight: bold">DETAILS OF THE TASK:</p>
```

Comment:
Copy details of the task from the `te-submission-email` according to my comments there.